lib/util/cache/repository/impl/s3.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { Readable } from 'stream';
+import {
+  GetObjectCommand,
+  GetObjectCommandInput,
+  PutObjectCommand,
+  PutObjectCommandInput,
+  PutObjectCommandOutput,
+  S3Client,
+} from '@aws-sdk/client-s3';
+import { mockClient } from 'aws-sdk-client-mock';
+import { partial } from '../../../../../test/util';
+import { GlobalConfig } from '../../../../config/global';
+import { logger } from '../../../../logger';
+import { parseS3Url } from '../../../s3';
+import * as s3Wrapper from '../../../s3';
+import type { RepoCacheRecord } from '../types';
+import { CacheFactory } from './cache-factory';
+import { RepoCacheS3 } from './s3';
+
+function createGetObjectCommandInput(
+  repository: string,
+  url: string
+): GetObjectCommandInput {
+  const platform = GlobalConfig.get('platform') ?? 'github';
```

Comment:
don't you set this to ' github'  for all tests? so we can hardcode it?

Reply:
yep, though to make it more general for future use